vtex/loaders/legacy/productListingPage.ts

```diff
@@ -177,8 +178,33 @@ const loader = async (
       props.similars ? withIsSimilarTo(req, ctx, product) : product
     ),
   );
+
+  // Get categories of the current department/category
+  const getCategoryFacets = (CategoriesTrees: LegacyFacet[]): LegacyFacet[] => {
+    const isDepartmentOrCategoryPage = !pageType;
```

Comment:
Is this condition accurate?

Reply:
yes, because if the page is not a departament or category, an error will occur

Reply:
@lucis